src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -493,19 +493,14 @@
         <PackageVersion>$(PackageVersion)</PackageVersion>
       </PackageFile>
 
-      <!-- Include Sources. Deduplicate so the nuspec doesn't contain multiple entries for each source file. -->
-      <PackageSources Include="@(File)"
-                      KeepMetadata="TargetPath;IsSourceCodeFile"
-                      KeepDuplicates="false"
-                      Condition="'%(File.IsSourceCodeFile)'=='true'" />
-      <!-- Add a placeholder source file if there are symbols but no sources. -->
-      <PackageSymbolFiles Include="@(File)" Condition="'%(File.IsSymbolFile)'=='true'" />
+      <!-- Add a placeholder source file if there are symbols but no sources, so that MyGet will treat symbol packages as valid -->
       <PackageSources Include="$(PlaceholderFile)" Condition="'@(PackageSymbolFiles)'!='' AND '@(PackageSources)'==''">
         <IsSourceCodeFile>true</IsSourceCodeFile>
```

Comment:
@ericstj @dagood Do we still need the IsSourceCodeFile metadata?

Reply:
I imagine it might be used in some later target to avoid adding sources to the main package, but I don't know the sequence well enough to know without poking around. I think @wtgodbe might be in the best position to answer that with actual build results.

Reply:
I would think we do need that metaData for the sake of excluding source code files, for example https://github.com/dotnet/buildtools/pull/2090/files#diff-03c502c8604ae1b4ae2efeb1f3d0c45eL490

Reply:
Sounds good I thought we removed all usages of the metadata. I'm good keeping it though.